Automattic/jetpack
# Boost: Update filenames

## Proposed changes:

* Rename the class files to pascle case

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:





Check all boost functionality and make sure everything is still working as expected
